### ISSUE TITLE
Question about the monotonicity loss

### ISSUE THREAD

**Issue by Keee03** [wang-fujin/PINN4SOH#1]
In Model/Model.py 

```
def train_one_epoch(self,epoch,dataloader):
    self.train()
    ...
    for iter,(x1,x2,y1,y2) in enumerate(dataloader):
        x1,x2,y1,y2 = x1.to(device),x2.to(device),y1.to(device),y2.to(device)
        u1,f1 = self.forward(x1)
        u2,f2 = self.forward(x2)
    
        # data loss
        loss1 = 0.5*self.loss_func(u1,y1) + 0.5*self.loss_func(u2,y2)
    
        # PDE loss
        f_target = torch.zeros_like(f1)
        loss2 = 0.5*self.loss_func(f1,f_target) + 0.5*self.loss_func(f2,f_target)
    
        # physics loss  y2-y1<0
        loss3 = self.relu(y2-y1).sum()
```
The definition of monotonicity loss in the article is:
![image](https://github.com/wang-fujin/PINN4SOH/assets/54928942/06101c21-f0e9-4f7b-b79f-571f4f431263)

In the code, y1 and y2 are actually the real SOH of two adjacent clips. However, according to the definition in the article, monotonicity loss should be the difference of estimated SOH. I would like to ask if there is something wrong with the code or my understanding about the monotonicity loss.

**Comment by wang-fujin** [comment]
Your understanding is correct. There was an error when we were organizing and uploading the code. It has now been fixed. Thank you for bringing this issue to our attention.
